src/shared/constants/status.js

```diff
@@ -73,14 +73,14 @@ export const MILSTATUS = [
   },
   {
     value: 'spouse',
-    label: 'Spouse / Family Member of Service Member'
+    label: 'Military Spouse / Dependent'
```

Comment:
We already cover `Military Spouse / Dependent` above, don't we?

Reply:
The above is a boolean re. military/civilian. This allows for individual categories of military status, i.e. spouse / servicemember.

Reply:
Why say military spouse isn't that redundant?

Reply:
Spouse / Dependent

Reply:
Can do